packages/palette/src/elements/Select/Select.tsx

```diff
@@ -30,102 +30,112 @@ export interface SelectProps
 }
 
 /** A drop-down select menu */
-export const Select: React.FC<SelectProps> = ({
-  description,
-  disabled,
-  error,
-  focus,
-  hover,
-  id,
-  name,
-  options,
-  required,
-  selected,
-  title,
-  variant = "default",
-  onSelect,
-  ...rest
-}) => {
-  const [boxProps, selectProps] = splitBoxProps(rest)
-
-  return (
-    <Box width="100%" {...boxProps}>
-      <Flex
-        as="label"
-        {...(variant === "inline"
-          ? {
-              flexDirection: "row",
-              alignItems: "center",
-            }
-          : {
-              flexDirection: "column",
-              alignItems: "flex-start",
-            })}
-        {...(id ? { for: id } : {})}
-      >
-        <div>
-          {title && (
-            <Text
-              variant="xs"
-              lineHeight={
-                variant === "inline" && description === undefined
-                  ? 1
-                  : undefined
+export const Select: ForwardRefExoticComponent<
+  SelectProps & { ref?: Ref<HTMLElement> }
```

Comment:
`Ref<HTMLSelectElement>`; I believe the Button example is looser because we change it using the `as` prop. (This is also probably fine; depending on your usecase)